graphprot/NeuralNet.py

```diff
@@ -487,28 +507,35 @@ def get_metrics(self, data='eval', threshold=4, binary=True):
             if len(self.valid_out) == 0:
```

Comment:
is `valid_out` properly initialized in case we don't have a validation set ? I can't find that ... 
same for `test_out` and `train_out`. If they are properly initialized just ignore that comment